src/stylesheets/components/_breadcrumb.scss

```diff
@@ -109,8 +109,15 @@ $breadcrumb-back-icon-aspect: (
         );
 
         @include u-display("inline-block");
-        @include u-padding-bottom($theme-breadcrumb-spacing-bottom);
-        @include u-padding-top($theme-breadcrumb-spacing-top);
+        @include u-padding-bottom($theme-breadcrumb-padding-bottom);
+        @include u-padding-top($theme-breadcrumb-padding-top);
+
+        // Prevent underline that extends beyond text
+        @include u-text("no-underline");
```

Comment:
Good eye! 

The hover/active styles from `@mixin button-unstyled` seems to be ignoring the `no-underline` on mobile.

[_button-unstyled.scss](https://github.com/uswds/uswds/blob/bc7ca86a5de23227a60382ae44a52bd02e5e1b8e/src/stylesheets/core/mixins/_button-unstyled.scss#L12)
```scss
@mixin button-unstyled {
  @include no-knockout-font-smoothing;
  @include typeset-link;
  background-color: transparent;
  border: 0;
  border-radius: 0;
  box-shadow: none;
  font-weight: font-weight("normal");
  margin: 0;
  padding: 0;
  text-align: left;

  &:hover,
  &:active {
    @include no-knockout-font-smoothing;
    background-color: transparent;
    box-shadow: none;
    text-decoration: underline;
  }
}

```

Normal:
![image](https://user-images.githubusercontent.com/3385219/86972261-4947e800-c138-11ea-850f-173876ceddc2.png)

Hover:
![image](https://user-images.githubusercontent.com/3385219/86972283-52d15000-c138-11ea-81b2-c6345eb743a1.png)